src/screens/transaction-details/components/DoubleLineTransactionDetailsRow.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { Box, Column, Columns, Inline, Stack, Text } from "@/design-system";
+
+type Props = {
+  leftComponent: React.ReactNode;
+  secondaryValue?: string;
+  title: string;
+  value: string;
+};
+
+export const DoubleLineTransactionDetailsRow: React.FC<Props> = ({
+  leftComponent,
+  secondaryValue,
+  value,
+  title,
+}) => (
+  <Box paddingVertical="20px">
+    <Columns space="10px" alignVertical="center">
+      <Column width="content">{leftComponent}</Column>
+      <Stack space="10px">
```

Comment:
not anything necessarily wrong here but just wondering if you know the impact of wrapping a component inside `<Columns>` with `<Column></Column>` or not?

Reply:
Wasn't it that Column changed how the width of columns behaves? At least I derived that idea from the docs